src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,167 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_PASSWORD_LENGTH = 6;
+    private static final int VALID_AGE = 18;
+    private static final String VALID_LOGIN = "Bob";
+    private static final String VALID_PASSWORD = "*".repeat(VALID_PASSWORD_LENGTH);
+    private static final String EXCEPTION = InvalidDataException.class.toString();
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    void register_UserWithValidCredentialsAddedToStorage_Ok() {

```

Comment:
```suggestion
    void register_userWithValidCredentialsAddedToStorage_ok() {
```
the same for other methods